testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/modcluster/StartWorkersInSuspendedModeTestCase.java

```diff
@@ -109,17 +116,28 @@ private void assertWorkerNodeContextIsStopped() throws Exception {
         Assert.assertEquals("stopped", modelNode.get(RESULT).get(STATUS).asString());
     }
 
-    static class ServerSetupTask extends CLIServerSetupTask {
+    static class ServerSetupTask extends ManagementServerSetupTask {
         public ServerSetupTask() {
-            this.builder
-               .node(NODE_1)
-               .setup("/subsystem=modcluster/proxy=default:write-attribute(name=advertise,value=false)")
-               .setup("/socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=proxy1:add(host=localhost,port=8590)")
-               .setup("/subsystem=modcluster/proxy=default:list-add(name=proxies,value=proxy1)")
-               .teardown("/subsystem=modcluster/proxy=default:list-remove(name=proxies,value=proxy1)")
-               .teardown("/socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=proxy1:remove()")
-               .teardown("/subsystem=modcluster/proxy=default:write-attribute(name=advertise,value=true)")
-            ;
+            super(createContainerSetConfigurationBuilder()
+                    .addContainer(NODE_1, createContainerConfigurationBuilder()
```

Comment:
There's a super constructor for a single container configuration, e.g.
super(NODE_1, createContainerConfigurationBuilder()...build());